.gitignore

```diff
@@ -1,3 +1,4 @@
 cmd/do-csi-plugin/do-csi-plugin
 .idea/
 *.sedbak
+.DS_Store
```

Comment:
Could you drop this one again please? IMO it should be in your global .gitignore.

Reply:
Actually, given that we also include `.idea`, I'm also okay with keeping it.